### ISSUE TITLE
Incorrect configuration suggestion when the word lovelace is within the repository.

### ISSUE THREAD

**Issue by peternijssen** [custom-components/hacs#47]
## Version of HACS
HACS version: 0.4.3 
Home Assistant version: 0.93.2

## Describe the bug
So according to this [documentation](https://custom-components.github.io/hacs/#note-for-plugin-developers) I need to remove the word "Lovelace" from the dist file when my repository starts with Lovelace.

So, that's exactly what I did:
https://github.com/peternijssen/lovelace-postnl-card/tree/master/dist

That's all working fine, except HACS gives me instructions to install this component like this:
```
When you add this to your configuration use this: 
  - url: /community_plugin/lovelace-postnl-card/lovelace-postnl-card.js
    type: js
```

This doesn't work, as it should be:
```
When you add this to your configuration use this: 
  - url: /community_plugin/lovelace-postnl-card/postnl-card.js
    type: js
```
So it seems it can't correctly cope with repositories starting with "lovelace".

**Comment by ludeeus** [comment]
A fix for that is coming later today.

**Comment by ludeeus** [comment]
This has been corrected in [`0.4.4`](https://github.com/custom-components/hacs/releases/tag/0.4.4)
